app/views/layouts/application.html.erb

```diff
@@ -14,7 +14,9 @@
   <body>
     <%= render "layouts/header" %>
     <main class="container mx-auto mt-4 mb-28 px-5 flex">
-      <%= render "layouts/flash" %>
+      <div id="flash">
```

Comment:
LRA-431 and LRA-444. Be sure to update or manage the JIRA tickets with comments

Reply:
LRA-431 and LRA-444 updated with a comment.